content/recipes/hot-reload.md

```diff
@@ -37,11 +36,10 @@ module.exports = function (options, webpack) {
     ],
     plugins: [
       ...options.plugins,
-      new webpack.HotModuleReplacementPlugin(),
       new webpack.WatchIgnorePlugin({
         paths: [/\.js$/, /\.d\.ts$/],
       }),
-      new RunScriptWebpackPlugin({ name: options.output.filename }),
```

Comment:
Any reason why we're removing the `RunScriptWebpackPlugin`? Without it, app won't start upon successful compilation (and that's what we claim in the docs)

Reply:
Docs suggests using `nest build` in this case though

Reply:
Oh! Thanks @kamilmysliwiec 😄 

I'm not used to use HMR. I'm going to close this issue and reply to the user! Thanks for the explanation